parm/config/config.resources

```diff
@@ -67,72 +73,88 @@ elif [ ${step} = "waveinit" ]; then
+    export memory_waveprep_gfs="220GB"
 
 elif [ ${step} = "wavepostsbs" ]; then
 
-    export wtime_wavepostsbs="06:00:00"
-    export npe_wavepostsbs=10
+    export wtime_wavepostsbs="00:20:00"
+    export wtime_wavepostsbs_gfs="03:00:00"
+    export npe_wavepostsbs=8
     export nth_wavepostsbs=1
-    export npe_node_wavepostsbs=$(echo "${npe_node_max} / ${nth_wavepostsbs}" | bc)
+    export npe_node_wavepostsbs=$npe_wavepostsbs
     export NTASKS=${npe_wavepostsbs}
+    export memory_wavepostsbs="10GB"
+    export memory_wavepostsbs_gfs="40GB"
 
 elif [ ${step} = "wavepostbndpnt" ]; then
 
-    export wtime_wavepostbndpnt="02:00:00"
-    export npe_wavepostbndpnt=280
+    export wtime_wavepostbndpnt="01:00:00"
+    export npe_wavepostbndpnt=240
     export nth_wavepostbndpnt=1
-    export npe_node_wavepostbndpnt=$(echo "${npe_node_max} / ${nth_wavepostbndpnt}" | bc)
+    export npe_node_wavepostbndpnt=80
     export NTASKS=${npe_wavepostbndpnt}
+    export is_exclusive=True
 
 elif [ ${step} = "wavepostbndpntbll" ]; then
 
     export wtime_wavepostbndpntbll="01:00:00"
-    export npe_wavepostbndpntbll=280
+    export npe_wavepostbndpntbll=448
     export nth_wavepostbndpntbll=1
-    export npe_node_wavepostbndpntbll=$(echo "${npe_node_max} / ${nth_wavepostbndpntbll}" | bc)
+    export npe_node_wavepostbndpntbll=112
     export NTASKS=${npe_wavepostbndpntbll}
+    export is_exclusive=True
 
 elif [ ${step} = "wavepostpnt" ]; then
 
-    export wtime_wavepostpnt="02:00:00"
-    export npe_wavepostpnt=280
+    export wtime_wavepostpnt="01:30:00"
+    export npe_wavepostpnt=200
     export nth_wavepostpnt=1
-    export npe_node_wavepostpnt=$(echo "${npe_node_max} / ${nth_wavepostpnt}" | bc)
+    export npe_node_wavepostpnt=50
+    if [[ "${npe_node_wavepostpnt}" -gt "${npe_node_max}" ]]; then export npe_node_wavepostpnt=${npe_node_max} ; fi
     export NTASKS=${npe_wavepostpnt}
+    export is_exclusive=True
 
 elif [ ${step} = "wavegempak" ]; then
 
-    export wtime_wavegempak="01:00:00"
-    export npe_wavegempak=${npe_node_max}
+    export wtime_wavegempak="02:00:00"
+    export npe_wavegempak=1
     export nth_wavegempak=1
-    export npe_node_wavegempak=$(echo "${npe_node_max} / ${nth_wavegempak}" | bc)
+    export npe_node_wavegempak=$npe_wavegempak
     export NTASKS=${npe_wavegempak}
+    export memory_wavegempak="10GB"
 
 elif [ ${step} = "waveawipsbulls" ]; then
 
-    export wtime_waveawipsbulls="00:30:00"
-    export npe_waveawipsbulls=${npe_node_max}
+    export wtime_waveawipsbulls="00:20:00"
+    export npe_waveawipsbulls=1
     export nth_waveawipsbulls=1
     export npe_node_waveawipsbulls=$(echo "${npe_node_max} / ${nth_waveawipsbulls}" | bc)
     export NTASKS=${npe_waveawipsbulls}
+    export is_exclusive=True
 
 elif [ ${step} = "waveawipsgridded" ]; then
 
-    export wtime_waveawipsgridded="00:30:00"
-    export npe_waveawipsgridded=${npe_node_max}
+    export wtime_waveawipsgridded="02:00:00"
```

Comment:
that is a huge increase.

Reply:
This was a change from GFSv16 ops: https://github.com/NOAA-EMC/global-workflow/blob/operations/ecf/scripts/gfs/wave/post/jgfs_wave_prdgen_gridded.ecf#L6